packages/vue-components/src/Overlay.vue

```diff
@@ -65,30 +49,20 @@ export default {
         publish('closeOverlay');
         // to prevent scrolling of the body when overlay is overscrolled
         document.body.style.overflow = 'hidden';
+        this.show = true;
+
+        this.$nextTick(() => {
+          $(this.$refs.navMenuContainer).find('a').on('click', this.toggleNavMenu);
+        });
```

Comment:
I've also noticed this strange behaviour where after clicking on a page nav's link, the overlay does not close.

For example on the [Using Components](https://deploy-preview-1556--markbind-master.netlify.app/userguide/usingcomponents) page:

[](https://user-images.githubusercontent.com/47470981/116770225-6e656880-aa74-11eb-9d3c-dd3cda9bb8dc.mp4)

Notice how after clicking on some of the links, the url changes but the overlay does not close (only after 2-3 clicks then it closes). I'm not sure if this method is causing this issue.